src/codegen/compile_classical.ml

```diff
@@ -11628,6 +11628,15 @@ and compile_prim_invocation (env : E.t) ae p es at =
     SR.Vanilla,
     StableMem.get_mem_size env ^^ BigNum.from_word64 env
 
+  | OtherPrim "rts_in_install", [] -> (* classical specific *)
+    assert (not !Flags.enhanced_orthogonal_persistence);
+    SR.Vanilla,
+    StableMem.stable64_size env ^^
+    G.i (Test (Wasm_exts.Values.I64 I64Op.Eqz)) ^^
+    G.if1 I32Type
+      (Bool.lit true)
+      (Bool.lit false)
```

Comment:
See `Bool.vanilla_lit`
```suggestion
    G.i (Test (Wasm_exts.Values.I32 I32Op.Eqz))
```

Reply:
But only on 32-bit, right? I'd have to extend to 64-bit  in the eop compiler. Will look.

Reply:
Correct, I didn't touch the 64-bit backend.

Reply:
obsolete due to refactoring.